services/adminInviteService.js

```diff
@@ -0,0 +1,42 @@
+const pgp = require('pg-promise')({});
+
+const cn = `postgresql://${process.env.REACT_APP_DATABASE_USER}:${process.env.REACT_APP_DATABASE_PASSWORD}@${process.env.REACT_APP_DATABASE_HOST}:${process.env.REACT_APP_DATABASE_PORT}/${process.env.REACT_APP_DATABASE_NAME}?ssl=true`; // For pgp
+const db = pgp(cn);
```

Comment:
Can we use the connection created in the config file instead?

Reply:
done!